pkg/machine/qemu/machine.go

```diff
@@ -859,6 +859,9 @@ func (v *MachineVM) Remove(_ string, opts machine.RemoveOptions) (string, func()
 	return confirmationMessage, func() error {
 		for _, f := range files {
 			if err := os.Remove(f); err != nil {
+				if os.IsNotExist(err) {
```

Comment:
please use errors.Is()

Reply:
Why?

Reply:
> This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist). 

https://pkg.go.dev/os#IsNotExist

Reply:
fixed

Reply:
Since we are only checking an error returned by the os package, I don't see the IsNotExist function  as a problem.  But it is not worth arguing about.